qlib/rl/neural_trading_migration/backtest.py

```diff
@@ -0,0 +1,230 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Put these to `qlib.rl.contrib` or `qlib.contrib.rl`, as it looks temporary.

Reply:
Just rename `qlib.rl.neural_trading_migration` to `qlib.rl.contrib`?

Reply:
Sounds good.